CurseForgeBadge.py

```diff
@@ -13,14 +13,31 @@ def landing():
     return render_template('index.html')
 
 
+@app.route('/title/<project>.svg')
+@app.route('/title/<project>(<l_colour>).svg')
+@app.route('/title/<project>(<l_colour>-<r_colour>-<text_colour>-<shadow_colour>-<logo_colour>).svg')
+def title(project, l_colour='E04E14', r_colour='2D2D2D', text_colour='fff',
```

Comment:
Maybe we could support `prefix` and `suffix` here as well? Thinking of something like `Get <project name> today!` :D